{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FrlsYrOfVlOF",
        "outputId": "61ae8ea0-822f-4947-c38d-28f7411c1c92"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ],
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "b62j349mVrFL",
        "outputId": "afc320b3-2287-4ba2-93d1-6c8689b2a956"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "✅ Project folder structure created successfully.\n"
          ]
        }
      ],
      "source": [
        "# ✅ Step 1: Create a fresh project folder structure on Google Drive\n",
        "import os\n",
        "\n",
        "base_dir = \"/content/drive/MyDrive/Cancer_WES_Project\"\n",
        "subdirs = [\n",
        "    \"raw_fastq\",\n",
        "    \"reference\",\n",
        "    \"bam_files\",\n",
        "    \"annotation\",\n",
        "    \"vcf_files\",\n",
        "    \"results\"\n",
        "]\n",
        "\n",
        "for sub in subdirs:\n",
        "    os.makedirs(os.path.join(base_dir, sub), exist_ok=True)\n",
        "\n",
        "print(\"✅ Project folder structure created successfully.\")\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "84V3iT3QV6xW",
        "outputId": "4f4967e3-b831-4408-db69-0de8075a1c6c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "W: Skipping acquire of configured file 'main/source/Sources' as repository 'https://r2u.stat.illinois.edu/ubuntu jammy InRelease' does not seem to provide it (sources.list entry misspelt?)\n",
            "Extracting templates from packages: 100%\n",
            "Selecting previously unselected package libhtscodecs2:amd64.\n",
            "(Reading database ... 126284 files and directories currently installed.)\n",
            "Preparing to unpack .../00-libhtscodecs2_1.1.1-3_amd64.deb ...\n",
            "Unpacking libhtscodecs2:amd64 (1.1.1-3) ...\n",
            "Selecting previously unselected package libhts3:amd64.\n",
            "Preparing to unpack .../01-libhts3_1.13+ds-2build1_amd64.deb ...\n",
            "Unpacking libhts3:amd64 (1.13+ds-2build1) ...\n",
            "Selecting previously unselected package bcftools.\n",
            "Preparing to unpack .../02-bcftools_1.13-1_amd64.deb ...\n",
            "Unpacking bcftools (1.13-1) ...\n",
            "Selecting previously unselected package bedtools.\n",
            "Preparing to unpack .../03-bedtools_2.30.0+dfsg-2ubuntu0.1_amd64.deb ...\n",
            "Unpacking bedtools (2.30.0+dfsg-2ubuntu0.1) ...\n",
            "Selecting previously unselected package bwa.\n",
            "Preparing to unpack .../04-bwa_0.7.17-6_amd64.deb ...\n",
            "Unpacking bwa (0.7.17-6) ...\n",
            "Selecting previously unselected package default-jre-headless.\n",
            "Preparing to unpack .../05-default-jre-headless_2%3a1.11-72build2_amd64.deb ...\n",
            "Unpacking default-jre-headless (2:1.11-72build2) ...\n",
            "Selecting previously unselected package libxtst6:amd64.\n",
            "Preparing to unpack .../06-libxtst6_2%3a1.2.3-1build4_amd64.deb ...\n",
            "Unpacking libxtst6:amd64 (2:1.2.3-1build4) ...\n",
            "Selecting previously unselected package openjdk-11-jre:amd64.\n",
            "Preparing to unpack .../07-openjdk-11-jre_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n",
            "Unpacking openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Selecting previously unselected package default-jre.\n",
            "Preparing to unpack .../08-default-jre_2%3a1.11-72build2_amd64.deb ...\n",
            "Unpacking default-jre (2:1.11-72build2) ...\n",
            "Selecting previously unselected package libcommons-math3-java.\n",
            "Preparing to unpack .../09-libcommons-math3-java_3.6.1-3_all.deb ...\n",
            "Unpacking libcommons-math3-java (3.6.1-3) ...\n",
            "Selecting previously unselected package libjbzip2-java.\n",
            "Preparing to unpack .../10-libjbzip2-java_0.9.1-7_all.deb ...\n",
            "Unpacking libjbzip2-java (0.9.1-7) ...\n",
            "Selecting previously unselected package libapache-pom-java.\n",
            "Preparing to unpack .../11-libapache-pom-java_18-1_all.deb ...\n",
            "Unpacking libapache-pom-java (18-1) ...\n",
            "Selecting previously unselected package libcommons-parent-java.\n",
            "Preparing to unpack .../12-libcommons-parent-java_43-1_all.deb ...\n",
            "Unpacking libcommons-parent-java (43-1) ...\n",
            "Selecting previously unselected package libcommons-compress-java.\n",
            "Preparing to unpack .../13-libcommons-compress-java_1.21-1_all.deb ...\n",
            "Unpacking libcommons-compress-java (1.21-1) ...\n",
            "Selecting previously unselected package libcommons-logging-java.\n",
            "Preparing to unpack .../14-libcommons-logging-java_1.2-2_all.deb ...\n",
            "Unpacking libcommons-logging-java (1.2-2) ...\n",
            "Selecting previously unselected package libcommons-jexl2-java.\n",
            "Preparing to unpack .../15-libcommons-jexl2-java_2.1.1-5_all.deb ...\n",
            "Unpacking libcommons-jexl2-java (2.1.1-5) ...\n",
            "Selecting previously unselected package libjson-simple-java.\n",
            "Preparing to unpack .../16-libjson-simple-java_2.3.0-1_all.deb ...\n",
            "Unpacking libjson-simple-java (2.3.0-1) ...\n",
            "Selecting previously unselected package libngs-sdk2:amd64.\n",
            "Preparing to unpack .../17-libngs-sdk2_2.11.2-3_amd64.deb ...\n",
            "Unpacking libngs-sdk2:amd64 (2.11.2-3) ...\n",
            "Selecting previously unselected package libngs-sdk-dev:amd64.\n",
            "Preparing to unpack .../18-libngs-sdk-dev_2.11.2-3_amd64.deb ...\n",
            "Unpacking libngs-sdk-dev:amd64 (2.11.2-3) ...\n",
            "Selecting previously unselected package libngs-java:amd64.\n",
            "Preparing to unpack .../19-libngs-java_2.11.2-3_amd64.deb ...\n",
            "Unpacking libngs-java:amd64 (2.11.2-3) ...\n",
            "Selecting previously unselected package libsnappy-jni.\n",
            "Preparing to unpack .../20-libsnappy-jni_1.1.8.3-1_amd64.deb ...\n",
            "Unpacking libsnappy-jni (1.1.8.3-1) ...\n",
            "Selecting previously unselected package libsnappy-java.\n",
            "Preparing to unpack .../21-libsnappy-java_1.1.8.3-1_all.deb ...\n",
            "Unpacking libsnappy-java (1.1.8.3-1) ...\n",
            "Selecting previously unselected package libhtsjdk-java.\n",
            "Preparing to unpack .../22-libhtsjdk-java_2.24.1+dfsg-1_all.deb ...\n",
            "Unpacking libhtsjdk-java (2.24.1+dfsg-1) ...\n",
            "Selecting previously unselected package libfindbin-libs-perl.\n",
            "Preparing to unpack .../23-libfindbin-libs-perl_2.190.02-1_all.deb ...\n",
            "Unpacking libfindbin-libs-perl (2.190.02-1) ...\n",
            "Selecting previously unselected package libargs4j-java.\n",
            "Preparing to unpack .../24-libargs4j-java_2.33-1.1_all.deb ...\n",
            "Unpacking libargs4j-java (2.33-1.1) ...\n",
            "Selecting previously unselected package libcommons-io-java.\n",
            "Preparing to unpack .../25-libcommons-io-java_2.11.0-2_all.deb ...\n",
            "Unpacking libcommons-io-java (2.11.0-2) ...\n",
            "Selecting previously unselected package libcommons-lang3-java.\n",
            "Preparing to unpack .../26-libcommons-lang3-java_3.11-1_all.deb ...\n",
            "Unpacking libcommons-lang3-java (3.11-1) ...\n",
            "Selecting previously unselected package libsis-base-jni.\n",
            "Preparing to unpack .../27-libsis-base-jni_18.09~pre1+git20180928.45fbd31+dfsg-3_amd64.deb ...\n",
            "Unpacking libsis-base-jni (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n",
            "Selecting previously unselected package libsis-base-java.\n",
            "Preparing to unpack .../28-libsis-base-java_18.09~pre1+git20180928.45fbd31+dfsg-3_all.deb ...\n",
            "Unpacking libsis-base-java (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n",
            "Selecting previously unselected package libsis-jhdf5-jni.\n",
            "Preparing to unpack .../29-libsis-jhdf5-jni_19.04.0+dfsg-4_amd64.deb ...\n",
            "Unpacking libsis-jhdf5-jni (19.04.0+dfsg-4) ...\n",
            "Selecting previously unselected package libsis-jhdf5-java.\n",
            "Preparing to unpack .../30-libsis-jhdf5-java_19.04.0+dfsg-4_all.deb ...\n",
            "Unpacking libsis-jhdf5-java (19.04.0+dfsg-4) ...\n",
            "Selecting previously unselected package fastqc.\n",
            "Preparing to unpack .../31-fastqc_0.11.9+dfsg-5_all.deb ...\n",
            "Unpacking fastqc (0.11.9+dfsg-5) ...\n",
            "Selecting previously unselected package fonts-dejavu-core.\n",
            "Preparing to unpack .../32-fonts-dejavu-core_2.37-2build1_all.deb ...\n",
            "Unpacking fonts-dejavu-core (2.37-2build1) ...\n",
            "Selecting previously unselected package fonts-dejavu-extra.\n",
            "Preparing to unpack .../33-fonts-dejavu-extra_2.37-2build1_all.deb ...\n",
            "Unpacking fonts-dejavu-extra (2.37-2build1) ...\n",
            "Selecting previously unselected package libxxf86dga1:amd64.\n",
            "Preparing to unpack .../34-libxxf86dga1_2%3a1.1.5-0ubuntu3_amd64.deb ...\n",
            "Unpacking libxxf86dga1:amd64 (2:1.1.5-0ubuntu3) ...\n",
            "Selecting previously unselected package x11-utils.\n",
            "Preparing to unpack .../35-x11-utils_7.7+5build2_amd64.deb ...\n",
            "Unpacking x11-utils (7.7+5build2) ...\n",
            "Selecting previously unselected package libatk-wrapper-java.\n",
            "Preparing to unpack .../36-libatk-wrapper-java_0.38.0-5build1_all.deb ...\n",
            "Unpacking libatk-wrapper-java (0.38.0-5build1) ...\n",
            "Selecting previously unselected package libatk-wrapper-java-jni:amd64.\n",
            "Preparing to unpack .../37-libatk-wrapper-java-jni_0.38.0-5build1_amd64.deb ...\n",
            "Unpacking libatk-wrapper-java-jni:amd64 (0.38.0-5build1) ...\n",
            "Selecting previously unselected package samtools.\n",
            "Preparing to unpack .../38-samtools_1.13-4_amd64.deb ...\n",
            "Unpacking samtools (1.13-4) ...\n",
            "Setting up libhtscodecs2:amd64 (1.1.1-3) ...\n",
            "Setting up libhts3:amd64 (1.13+ds-2build1) ...\n",
            "Setting up default-jre-headless (2:1.11-72build2) ...\n",
            "Setting up bwa (0.7.17-6) ...\n",
            "Setting up libxtst6:amd64 (2:1.2.3-1build4) ...\n",
            "Setting up bcftools (1.13-1) ...\n",
            "Setting up libngs-sdk2:amd64 (2.11.2-3) ...\n",
            "Setting up libxxf86dga1:amd64 (2:1.1.5-0ubuntu3) ...\n",
            "Setting up openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Setting up libfindbin-libs-perl (2.190.02-1) ...\n",
            "Setting up default-jre (2:1.11-72build2) ...\n",
            "Setting up libjson-simple-java (2.3.0-1) ...\n",
            "Setting up samtools (1.13-4) ...\n",
            "Setting up libsis-base-jni (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n",
            "Setting up libapache-pom-java (18-1) ...\n",
            "Setting up fonts-dejavu-core (2.37-2build1) ...\n",
            "Setting up libjbzip2-java (0.9.1-7) ...\n",
            "Setting up libngs-sdk-dev:amd64 (2.11.2-3) ...\n",
            "Setting up fonts-dejavu-extra (2.37-2build1) ...\n",
            "Setting up x11-utils (7.7+5build2) ...\n",
            "Setting up libcommons-math3-java (3.6.1-3) ...\n",
            "Setting up libsis-jhdf5-jni (19.04.0+dfsg-4) ...\n",
            "Setting up libatk-wrapper-java (0.38.0-5build1) ...\n",
            "Setting up libngs-java:amd64 (2.11.2-3) ...\n",
            "Setting up libsnappy-jni (1.1.8.3-1) ...\n",
            "Setting up bedtools (2.30.0+dfsg-2ubuntu0.1) ...\n",
            "Setting up libargs4j-java (2.33-1.1) ...\n",
            "Setting up libcommons-parent-java (43-1) ...\n",
            "Setting up libcommons-logging-java (1.2-2) ...\n",
            "Setting up libatk-wrapper-java-jni:amd64 (0.38.0-5build1) ...\n",
            "Setting up libcommons-jexl2-java (2.1.1-5) ...\n",
            "Setting up libcommons-lang3-java (3.11-1) ...\n",
            "Setting up libsnappy-java (1.1.8.3-1) ...\n",
            "Setting up libcommons-compress-java (1.21-1) ...\n",
            "Setting up libcommons-io-java (2.11.0-2) ...\n",
            "Setting up libsis-base-java (18.09~pre1+git20180928.45fbd31+dfsg-3) ...\n",
            "Setting up libhtsjdk-java (2.24.1+dfsg-1) ...\n",
            "Setting up libsis-jhdf5-java (19.04.0+dfsg-4) ...\n",
            "Setting up fastqc (0.11.9+dfsg-5) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for mailcap (3.70+nmu1ubuntu1) ...\n",
            "Processing triggers for fontconfig (2.13.1-4.2ubuntu5) ...\n",
            "Processing triggers for hicolor-icon-theme (0.17-2) ...\n",
            "Processing triggers for libc-bin (2.35-0ubuntu3.8) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm_debug.so.1 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_opencl.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libumf.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_loader.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm.so.1 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libhwloc.so.15 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero_v2.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n",
            "\n"
          ]
        }
      ],
      "source": [
        "# ✅ Install core bioinformatics tools\n",
        "!apt-get update -qq\n",
        "!apt-get install -y -qq fastqc bwa samtools bcftools bedtools\n",
        "!wget -q https://github.com/broadinstitute/gatk/releases/download/4.4.0.0/gatk-4.4.0.0.zip\n",
        "!unzip -q gatk-4.4.0.0.zip\n",
        "!mv gatk-4.4.0.0 /opt/gatk\n",
        "!chmod +x /opt/gatk/gatk\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4cBxDQoyV-k6",
        "outputId": "aae29b5a-72a8-4b47-ba5e-214f8730c4ee"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "FastQC v0.11.9\n",
            "\n",
            "samtools 1.13\n",
            "Using GATK jar /opt/gatk/gatk-package-4.4.0.0-local.jar\n",
            "Running:\n",
            "    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /opt/gatk/gatk-package-4.4.0.0-local.jar --version\n",
            "Error: LinkageError occurred while loading main class org.broadinstitute.hellbender.Main\n",
            "\tjava.lang.UnsupportedClassVersionError: org/broadinstitute/hellbender/Main has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0\n",
            "bcftools 1.13\n"
          ]
        }
      ],
      "source": [
        "!fastqc --version\n",
        "!bwa 2>&1 | head -n 1\n",
        "!samtools --version | head -n 1\n",
        "!/opt/gatk/gatk --version\n",
        "!bcftools --version | head -n 1\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ZRIX5556WQwz",
        "outputId": "d4a3242a-1005-4e2d-ec20-4012ebbef844"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following packages were automatically installed and are no longer required:\n",
            "  fonts-dejavu-extra java-common libapache-pom-java libargs4j-java\n",
            "  libatk-wrapper-java libatk-wrapper-java-jni libcommons-compress-java\n",
            "  libcommons-io-java libcommons-jexl2-java libcommons-lang3-java\n",
            "  libcommons-logging-java libcommons-math3-java libcommons-parent-java\n",
            "  libfindbin-libs-perl libhtsjdk-java libjbzip2-java libjson-simple-java\n",
            "  libngs-java libngs-sdk-dev libngs-sdk2 libpcsclite1 libsis-base-java\n",
            "  libsis-base-jni libsis-jhdf5-java libsis-jhdf5-jni libsnappy-java\n",
            "  libsnappy-jni\n",
            "Use 'sudo apt autoremove' to remove them.\n",
            "The following packages will be REMOVED:\n",
            "  ca-certificates-java default-jre default-jre-headless fastqc\n",
            "  openjdk-11-jdk-headless openjdk-11-jre openjdk-11-jre-headless\n",
            "0 upgraded, 0 newly installed, 7 to remove and 35 not upgraded.\n",
            "After this operation, 260 MB disk space will be freed.\n",
            "(Reading database ... 127097 files and directories currently installed.)\n",
            "Removing fastqc (0.11.9+dfsg-5) ...\n",
            "Removing default-jre (2:1.11-72build2) ...\n",
            "Removing openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Removing openjdk-11-jdk-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Removing default-jre-headless (2:1.11-72build2) ...\n",
            "Removing openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Removing ca-certificates-java (20190909ubuntu1.2) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for ca-certificates (20240203~22.04.1) ...\n",
            "Updating certificates in /etc/ssl/certs...\n",
            "0 added, 0 removed; done.\n",
            "Running hooks in /etc/ca-certificates/update.d...\n",
            "\n",
            "updates of cacerts keystore disabled.\n",
            "done.\n",
            "Processing triggers for mailcap (3.70+nmu1ubuntu1) ...\n",
            "Processing triggers for hicolor-icon-theme (0.17-2) ...\n",
            "Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease\n",
            "Hit:2 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease\n",
            "Hit:3 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  InRelease\n",
            "Hit:4 http://archive.ubuntu.com/ubuntu jammy InRelease\n",
            "Hit:5 https://r2u.stat.illinois.edu/ubuntu jammy InRelease\n",
            "Hit:6 http://archive.ubuntu.com/ubuntu jammy-updates InRelease\n",
            "Hit:7 http://archive.ubuntu.com/ubuntu jammy-backports InRelease\n",
            "Hit:8 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease\n",
            "Hit:9 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease\n",
            "Hit:10 https://ppa.launchpadcontent.net/ubuntugis/ppa/ubuntu jammy InRelease\n",
            "Reading package lists... Done\n",
            "W: Skipping acquire of configured file 'main/source/Sources' as repository 'https://r2u.stat.illinois.edu/ubuntu jammy InRelease' does not seem to provide it (sources.list entry misspelt?)\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following packages were automatically installed and are no longer required:\n",
            "  fonts-dejavu-extra libapache-pom-java libargs4j-java libatk-wrapper-java\n",
            "  libatk-wrapper-java-jni libcommons-compress-java libcommons-io-java\n",
            "  libcommons-jexl2-java libcommons-lang3-java libcommons-logging-java\n",
            "  libcommons-math3-java libcommons-parent-java libfindbin-libs-perl\n",
            "  libhtsjdk-java libjbzip2-java libjson-simple-java libngs-java libngs-sdk-dev\n",
            "  libngs-sdk2 libsis-base-java libsis-base-jni libsis-jhdf5-java\n",
            "  libsis-jhdf5-jni libsnappy-java libsnappy-jni\n",
            "Use 'sudo apt autoremove' to remove them.\n",
            "The following additional packages will be installed:\n",
            "  ca-certificates-java\n",
            "Suggested packages:\n",
            "  libnss-mdns fonts-ipafont-gothic fonts-ipafont-mincho fonts-wqy-microhei\n",
            "  | fonts-wqy-zenhei fonts-indic\n",
            "The following NEW packages will be installed:\n",
            "  ca-certificates-java openjdk-17-jre-headless\n",
            "0 upgraded, 2 newly installed, 0 to remove and 35 not upgraded.\n",
            "Need to get 48.3 MB of archives.\n",
            "After this operation, 193 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 openjdk-17-jre-headless amd64 17.0.15+6~us1-0ubuntu1~22.04 [48.3 MB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 ca-certificates-java all 20190909ubuntu1.2 [12.1 kB]\n",
            "Fetched 48.3 MB in 2s (22.9 MB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78, <> line 2.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package openjdk-17-jre-headless:amd64.\n",
            "(Reading database ... 126565 files and directories currently installed.)\n",
            "Preparing to unpack .../openjdk-17-jre-headless_17.0.15+6~us1-0ubuntu1~22.04_amd64.deb ...\n",
            "Unpacking openjdk-17-jre-headless:amd64 (17.0.15+6~us1-0ubuntu1~22.04) ...\n",
            "Selecting previously unselected package ca-certificates-java.\n",
            "Preparing to unpack .../ca-certificates-java_20190909ubuntu1.2_all.deb ...\n",
            "Unpacking ca-certificates-java (20190909ubuntu1.2) ...\n",
            "Setting up openjdk-17-jre-headless:amd64 (17.0.15+6~us1-0ubuntu1~22.04) ...\n",
            "update-alternatives: using /usr/lib/jvm/java-17-openjdk-amd64/bin/java to provide /usr/bin/java (java) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-17-openjdk-amd64/bin/jpackage to provide /usr/bin/jpackage (jpackage) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-17-openjdk-amd64/bin/keytool to provide /usr/bin/keytool (keytool) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-17-openjdk-amd64/bin/rmiregistry to provide /usr/bin/rmiregistry (rmiregistry) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-17-openjdk-amd64/lib/jexec to provide /usr/bin/jexec (jexec) in auto mode\n",
            "Setting up ca-certificates-java (20190909ubuntu1.2) ...\n",
            "Processing triggers for ca-certificates (20240203~22.04.1) ...\n",
            "Updating certificates in /etc/ssl/certs...\n",
            "0 added, 0 removed; done.\n",
            "Running hooks in /etc/ca-certificates/update.d...\n",
            "\n",
            "done.\n",
            "done.\n",
            "openjdk version \"17.0.15\" 2025-04-15\n",
            "OpenJDK Runtime Environment (build 17.0.15+6-Ubuntu-0ubuntu122.04)\n",
            "OpenJDK 64-Bit Server VM (build 17.0.15+6-Ubuntu-0ubuntu122.04, mixed mode, sharing)\n"
          ]
        }
      ],
      "source": [
        "# Remove older Java versions if needed\n",
        "!sudo apt-get remove openjdk-11-jre-headless -y\n",
        "!sudo apt-get update\n",
        "\n",
        "# Install Java 17\n",
        "!sudo apt install openjdk-17-jre-headless -y\n",
        "\n",
        "# Verify version\n",
        "!java -version\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hqpy-OgtWWSJ",
        "outputId": "90c8d7c1-85d3-4301-bc96-95f14ae69a3b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using GATK jar /opt/gatk/gatk-package-4.4.0.0-local.jar\n",
            "Running:\n",
            "    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /opt/gatk/gatk-package-4.4.0.0-local.jar --version\n",
            "The Genome Analysis Toolkit (GATK) v4.4.0.0\n",
            "HTSJDK Version: 3.0.5\n",
            "Picard Version: 3.0.0\n"
          ]
        }
      ],
      "source": [
        "!/opt/gatk/gatk --version\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "sz1cor0qWcGR",
        "outputId": "901770a2-7979-4859-8af5-0466dfa6ee6d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following packages were automatically installed and are no longer required:\n",
            "  fonts-dejavu-extra libapache-pom-java libargs4j-java libatk-wrapper-java\n",
            "  libatk-wrapper-java-jni libcommons-compress-java libcommons-io-java\n",
            "  libcommons-jexl2-java libcommons-lang3-java libcommons-logging-java\n",
            "  libcommons-math3-java libcommons-parent-java libfindbin-libs-perl\n",
            "  libhtsjdk-java libjbzip2-java libjson-simple-java libngs-java libngs-sdk-dev\n",
            "  libngs-sdk2 libsis-base-java libsis-base-jni libsis-jhdf5-java\n",
            "  libsis-jhdf5-jni libsnappy-java libsnappy-jni\n",
            "Use 'apt autoremove' to remove them.\n",
            "The following additional packages will be installed:\n",
            "  blends-common libkdf5-2 libncbi-vdb2 libncbi-wvdb2 med-config menu\n",
            "Suggested packages:\n",
            "  blends-doc menu-l10n gksu | kde-runtime | ktsuss\n",
            "The following NEW packages will be installed:\n",
            "  blends-common libkdf5-2 libncbi-vdb2 libncbi-wvdb2 med-config menu\n",
            "  sra-toolkit\n",
            "0 upgraded, 7 newly installed, 0 to remove and 35 not upgraded.\n",
            "Need to get 8,290 kB of archives.\n",
            "After this operation, 23.0 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 menu amd64 2.1.47ubuntu4 [354 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/universe amd64 blends-common all 0.7.4ubuntu1 [15.9 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libkdf5-2 amd64 2.11.2+dfsg-4build2 [14.7 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libncbi-vdb2 amd64 2.11.2+dfsg-4build2 [1,364 kB]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libncbi-wvdb2 amd64 2.11.2+dfsg-4build2 [1,252 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu jammy/universe amd64 sra-toolkit amd64 2.11.3+dfsg-1ubuntu1 [5,277 kB]\n",
            "Get:7 http://archive.ubuntu.com/ubuntu jammy/universe amd64 med-config all 3.7.1 [11.6 kB]\n",
            "Fetched 8,290 kB in 0s (21.2 MB/s)\n",
            "Preconfiguring packages ...\n",
            "Selecting previously unselected package menu.\n",
            "(Reading database ... 126890 files and directories currently installed.)\n",
            "Preparing to unpack .../0-menu_2.1.47ubuntu4_amd64.deb ...\n",
            "Unpacking menu (2.1.47ubuntu4) ...\n",
            "Selecting previously unselected package blends-common.\n",
            "Preparing to unpack .../1-blends-common_0.7.4ubuntu1_all.deb ...\n",
            "Unpacking blends-common (0.7.4ubuntu1) ...\n",
            "Selecting previously unselected package libkdf5-2.\n",
            "Preparing to unpack .../2-libkdf5-2_2.11.2+dfsg-4build2_amd64.deb ...\n",
            "Unpacking libkdf5-2 (2.11.2+dfsg-4build2) ...\n",
            "Selecting previously unselected package libncbi-vdb2.\n",
            "Preparing to unpack .../3-libncbi-vdb2_2.11.2+dfsg-4build2_amd64.deb ...\n",
            "Unpacking libncbi-vdb2 (2.11.2+dfsg-4build2) ...\n",
            "Selecting previously unselected package libncbi-wvdb2.\n",
            "Preparing to unpack .../4-libncbi-wvdb2_2.11.2+dfsg-4build2_amd64.deb ...\n",
            "Unpacking libncbi-wvdb2 (2.11.2+dfsg-4build2) ...\n",
            "Selecting previously unselected package sra-toolkit.\n",
            "Preparing to unpack .../5-sra-toolkit_2.11.3+dfsg-1ubuntu1_amd64.deb ...\n",
            "Unpacking sra-toolkit (2.11.3+dfsg-1ubuntu1) ...\n",
            "Selecting previously unselected package med-config.\n",
            "Preparing to unpack .../6-med-config_3.7.1_all.deb ...\n",
            "Unpacking med-config (3.7.1) ...\n",
            "Setting up libncbi-vdb2 (2.11.2+dfsg-4build2) ...\n",
            "Setting up libkdf5-2 (2.11.2+dfsg-4build2) ...\n",
            "Setting up libncbi-wvdb2 (2.11.2+dfsg-4build2) ...\n",
            "Setting up menu (2.1.47ubuntu4) ...\n",
            "Setting up blends-common (0.7.4ubuntu1) ...\n",
            "Setting up sra-toolkit (2.11.3+dfsg-1ubuntu1) ...\n",
            "Setting up med-config (3.7.1) ...\n",
            "Adding group `med' (GID 107) ...\n",
            "Done.\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for libc-bin (2.35-0ubuntu3.8) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm_debug.so.1 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_opencl.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libumf.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_loader.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtcm.so.1 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libhwloc.so.15 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libur_adapter_level_zero_v2.so.0 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n",
            "\n",
            "Processing triggers for menu (2.1.47ubuntu4) ...\n",
            "/usr/bin/prefetch\n",
            "/usr/bin/fasterq-dump\n"
          ]
        }
      ],
      "source": [
        "# ✅ Install SRA Toolkit\n",
        "!apt-get install -y sra-toolkit\n",
        "\n",
        "# ✅ Confirm installation\n",
        "!which prefetch\n",
        "!which fasterq-dump\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ZccODzKiWe5F",
        "outputId": "fabee7ab-05c9-414a-b4b3-79ee8b3f2eb3"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/drive/MyDrive/Cancer_WES_Project/raw_fastq\n",
            "spots read      : 32,182,906\n",
            "reads read      : 64,365,812\n",
            "reads written   : 64,365,812\n"
          ]
        }
      ],
      "source": [
        "# ✅ Step: Go to the FASTQ folder\n",
        "%cd /content/drive/MyDrive/Cancer_WES_Project/raw_fastq\n",
        "\n",
        "# ✅ Step: Extract FASTQ with 4 threads\n",
        "!fasterq-dump SRR766039  -e 4\n",
        "\n",
        "# ✅ Step: Compress the FASTQ files\n",
        "!gzip SRR766039_1.fastq\n",
        "!gzip SRR766039_2.fastq\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RDhhynZbWish",
        "outputId": "bd5a0fdb-fcb0-498f-bd6f-ed969d78e290"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "W: Skipping acquire of configured file 'main/source/Sources' as repository 'https://r2u.stat.illinois.edu/ubuntu jammy InRelease' does not seem to provide it (sources.list entry misspelt?)\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  default-jre default-jre-headless openjdk-11-jre openjdk-11-jre-headless\n",
            "Suggested packages:\n",
            "  libnss-mdns fonts-ipafont-gothic fonts-ipafont-mincho fonts-wqy-microhei\n",
            "  | fonts-wqy-zenhei fonts-indic\n",
            "The following NEW packages will be installed:\n",
            "  default-jre default-jre-headless fastqc openjdk-11-jre\n",
            "  openjdk-11-jre-headless\n",
            "0 upgraded, 5 newly installed, 0 to remove and 35 not upgraded.\n",
            "Need to get 43.2 MB of archives.\n",
            "After this operation, 177 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openjdk-11-jre-headless amd64 11.0.28+6-1ubuntu1~22.04.1 [42.6 MB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 default-jre-headless amd64 2:1.11-72build2 [3,042 B]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 openjdk-11-jre amd64 11.0.28+6-1ubuntu1~22.04.1 [214 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 default-jre amd64 2:1.11-72build2 [896 B]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu jammy/universe amd64 fastqc all 0.11.9+dfsg-5 [370 kB]\n",
            "Fetched 43.2 MB in 2s (21.3 MB/s)\n",
            "Selecting previously unselected package openjdk-11-jre-headless:amd64.\n",
            "(Reading database ... 127240 files and directories currently installed.)\n",
            "Preparing to unpack .../openjdk-11-jre-headless_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n",
            "Unpacking openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Selecting previously unselected package default-jre-headless.\n",
            "Preparing to unpack .../default-jre-headless_2%3a1.11-72build2_amd64.deb ...\n",
            "Unpacking default-jre-headless (2:1.11-72build2) ...\n",
            "Selecting previously unselected package openjdk-11-jre:amd64.\n",
            "Preparing to unpack .../openjdk-11-jre_11.0.28+6-1ubuntu1~22.04.1_amd64.deb ...\n",
            "Unpacking openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Selecting previously unselected package default-jre.\n",
            "Preparing to unpack .../default-jre_2%3a1.11-72build2_amd64.deb ...\n",
            "Unpacking default-jre (2:1.11-72build2) ...\n",
            "Selecting previously unselected package fastqc.\n",
            "Preparing to unpack .../fastqc_0.11.9+dfsg-5_all.deb ...\n",
            "Unpacking fastqc (0.11.9+dfsg-5) ...\n",
            "Setting up openjdk-11-jre-headless:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/jjs to provide /usr/bin/jjs (jjs) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/rmid to provide /usr/bin/rmid (rmid) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/pack200 to provide /usr/bin/pack200 (pack200) in auto mode\n",
            "update-alternatives: using /usr/lib/jvm/java-11-openjdk-amd64/bin/unpack200 to provide /usr/bin/unpack200 (unpack200) in auto mode\n",
            "Setting up openjdk-11-jre:amd64 (11.0.28+6-1ubuntu1~22.04.1) ...\n",
            "Setting up default-jre-headless (2:1.11-72build2) ...\n",
            "Setting up default-jre (2:1.11-72build2) ...\n",
            "Setting up fastqc (0.11.9+dfsg-5) ...\n",
            "Processing triggers for mailcap (3.70+nmu1ubuntu1) ...\n",
            "Processing triggers for hicolor-icon-theme (0.17-2) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "FastQC v0.11.9\n"
          ]
        }
      ],
      "source": [
        "# ✅ Reinstall FastQC\n",
        "!apt-get update -qq\n",
        "!apt-get install -y fastqc\n",
        "!fastqc --version\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VofSoJQyhwRF",
        "outputId": "c51b985b-df86-4a21-b039-5c334a69fe59"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/drive/MyDrive/Cancer_WES_Project/raw_fastq\n",
            "Started analysis of SRR766039_1.fastq.gz\n",
            "Approx 5% complete for SRR766039_1.fastq.gz\n",
            "Approx 10% complete for SRR766039_1.fastq.gz\n",
            "Approx 15% complete for SRR766039_1.fastq.gz\n",
            "Approx 20% complete for SRR766039_1.fastq.gz\n",
            "Approx 25% complete for SRR766039_1.fastq.gz\n",
            "Approx 30% complete for SRR766039_1.fastq.gz\n",
            "Approx 35% complete for SRR766039_1.fastq.gz\n",
            "Approx 40% complete for SRR766039_1.fastq.gz\n",
            "Approx 45% complete for SRR766039_1.fastq.gz\n",
            "Approx 50% complete for SRR766039_1.fastq.gz\n",
            "Approx 55% complete for SRR766039_1.fastq.gz\n",
            "Approx 60% complete for SRR766039_1.fastq.gz\n",
            "Approx 65% complete for SRR766039_1.fastq.gz\n",
            "Approx 70% complete for SRR766039_1.fastq.gz\n",
            "Approx 75% complete for SRR766039_1.fastq.gz\n",
            "Approx 80% complete for SRR766039_1.fastq.gz\n",
            "Approx 85% complete for SRR766039_1.fastq.gz\n",
            "Approx 90% complete for SRR766039_1.fastq.gz\n",
            "Approx 95% complete for SRR766039_1.fastq.gz\n",
            "Analysis complete for SRR766039_1.fastq.gz\n",
            "Started analysis of SRR766039_2.fastq.gz\n",
            "Approx 5% complete for SRR766039_2.fastq.gz\n",
            "Approx 10% complete for SRR766039_2.fastq.gz\n",
            "Approx 15% complete for SRR766039_2.fastq.gz\n",
            "Approx 20% complete for SRR766039_2.fastq.gz\n",
            "Approx 25% complete for SRR766039_2.fastq.gz\n",
            "Approx 30% complete for SRR766039_2.fastq.gz\n",
            "Approx 35% complete for SRR766039_2.fastq.gz\n",
            "Approx 40% complete for SRR766039_2.fastq.gz\n",
            "Approx 45% complete for SRR766039_2.fastq.gz\n",
            "Approx 50% complete for SRR766039_2.fastq.gz\n",
            "Approx 55% complete for SRR766039_2.fastq.gz\n",
            "Approx 60% complete for SRR766039_2.fastq.gz\n",
            "Approx 65% complete for SRR766039_2.fastq.gz\n",
            "Approx 70% complete for SRR766039_2.fastq.gz\n",
            "Approx 75% complete for SRR766039_2.fastq.gz\n",
            "Approx 80% complete for SRR766039_2.fastq.gz\n",
            "Approx 85% complete for SRR766039_2.fastq.gz\n",
            "Approx 90% complete for SRR766039_2.fastq.gz\n",
            "Approx 95% complete for SRR766039_2.fastq.gz\n",
            "Analysis complete for SRR766039_2.fastq.gz\n"
          ]
        }
      ],
      "source": [
        "# ✅ Move into the FASTQ folder\n",
        "%cd /content/drive/MyDrive/Cancer_WES_Project/raw_fastq\n",
        "\n",
        "# ✅ Run FastQC on both files\n",
        "!fastqc SRR766039_1.fastq.gz SRR766039_2.fastq.gz\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "id": "8j0IYTj_h4xJ",
        "outputId": "87bb494e-eb7d-4bed-b2c6-ec91cd3f4ac5"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_d8026128-2d25-406f-b38e-e8efcf5fbdbb\", \"SRR766039_1_fastqc.html\", 629827)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_be30ff27-5d67-4502-97b8-aba84fe1b154\", \"SRR766039_2_fastqc.html\", 640653)"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "from google.colab import files\n",
        "\n",
        "# Download both reports\n",
        "files.download(\"/content/drive/MyDrive/Cancer_WES_Project/raw_fastq/SRR766039_1_fastqc.html\")\n",
        "files.download(\"/content/drive/MyDrive/Cancer_WES_Project/raw_fastq/SRR766039_2_fastqc.html\")\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 13,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "rb_hWfPgjKW5",
        "outputId": "aaa799b5-afd3-48c8-f4d7-3c6f616605cd"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/reference\n",
            "--2025-08-07 07:52:18--  https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.40_GRCh38.p14/GRCh38_major_release_seqs_for_alignment_pipelines/GCA_000001405.15_GRCh38_no_alt_plus_hs38d1_analysis_set.fna.gz\n",
            "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.13, 130.14.250.7, 130.14.250.10, ...\n",
            "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.13|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 874637925 (834M) [application/x-gzip]\n",
            "Saving to: ‘GRCh38_no_alt_plus_decoy.fa.gz’\n",
            "\n",
            "GRCh38_no_alt_plus_ 100%[===================>] 834.12M  45.2MB/s    in 17s     \n",
            "\n",
            "2025-08-07 07:52:35 (49.2 MB/s) - ‘GRCh38_no_alt_plus_decoy.fa.gz’ saved [874637925/874637925]\n",
            "\n",
            "-rw-r--r-- 1 root root 3.0G Feb 17  2016 /content/reference/GRCh38_no_alt_plus_decoy.fa\n"
          ]
        }
      ],
      "source": [
        "# Step 0: Delete old corrupted files (if any)\n",
        "!rm -f /content/reference/GRCh38_no_alt_plus_decoy.fa*\n",
        "!mkdir -p /content/reference\n",
        "%cd /content/reference\n",
        "\n",
        "# Step 1: Redownload cleanly (~834 MB compressed)\n",
        "!wget -O GRCh38_no_alt_plus_decoy.fa.gz \"https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.40_GRCh38.p14/GRCh38_major_release_seqs_for_alignment_pipelines/GCA_000001405.15_GRCh38_no_alt_plus_hs38d1_analysis_set.fna.gz\"\n",
        "\n",
        "# Step 2: Decompress (~10–11 GB uncompressed)\n",
        "!gunzip -f GRCh38_no_alt_plus_decoy.fa.gz\n",
        "\n",
        "# Step 3: Confirm proper decompressed size\n",
        "!ls -lh /content/reference/GRCh38_no_alt_plus_decoy.fa\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "SpaWwsp_jPHo",
        "outputId": "67f4c917-52fe-4023-a899-6fd9fa174952"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "44371205 /content/reference/GRCh38_no_alt_plus_decoy.fa\n",
            "2580\n",
            ">chr1  AC:CM000663.2  gi:568336023  LN:248956422  rl:Chromosome  M5:6aef897c3d6ff0c78aff06ac189178dd  AS:GRCh38\n",
            ">chr2  AC:CM000664.2  gi:568336022  LN:242193529  rl:Chromosome  M5:f98db672eb0993dcfdabafe2a882905c  AS:GRCh38\n",
            ">chr3  AC:CM000665.2  gi:568336021  LN:198295559  rl:Chromosome  M5:76635a41ea913a405ded820447d067b0  AS:GRCh38\n",
            ">chr4  AC:CM000666.2  gi:568336020  LN:190214555  rl:Chromosome  M5:3210fecf1eb92d5489da4346b3fddc6e  AS:GRCh38\n",
            ">chr5  AC:CM000667.2  gi:568336019  LN:181538259  rl:Chromosome  M5:a811b3dc9fe66af729dc0dddf7fa4f13  AS:GRCh38  hm:47309185-49591369\n",
            ">chr6  AC:CM000668.2  gi:568336018  LN:170805979  rl:Chromosome  M5:5691468a67c7e7a7b5f2a3a683792c29  AS:GRCh38\n",
            ">chr7  AC:CM000669.2  gi:568336017  LN:159345973  rl:Chromosome  M5:cc044cc2256a1141212660fb07b6171e  AS:GRCh38\n",
            ">chr8  AC:CM000670.2  gi:568336016  LN:145138636  rl:Chromosome  M5:c67955b5f7815a9a1edfaa15893d3616  AS:GRCh38\n",
            ">chr9  AC:CM000671.2  gi:568336015  LN:138394717  rl:Chromosome  M5:6c198acf68b5af7b9d676dfdd531b5de  AS:GRCh38\n",
            ">chr10  AC:CM000672.2  gi:568336014  LN:133797422  rl:Chromosome  M5:c0eeee7acfdaf31b770a509bdaa6e51a  AS:GRCh38\n"
          ]
        }
      ],
      "source": [
        "# Count number of lines\n",
        "!wc -l /content/reference/GRCh38_no_alt_plus_decoy.fa\n",
        "\n",
        "# Show how many sequences (chromosomes/contigs) are in the file\n",
        "!grep \"^>\" /content/reference/GRCh38_no_alt_plus_decoy.fa | wc -l\n",
        "\n",
        "# Print the first 10 sequence headers\n",
        "!grep \"^>\" /content/reference/GRCh38_no_alt_plus_decoy.fa | head -10\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7vivI_pckTHQ",
        "outputId": "f782d86b-6a74-4832-ae4a-8c153decc30c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/reference\n",
            "[bwa_index] Pack FASTA... 30.44 sec\n",
            "[bwa_index] Construct BWT for the packed sequence...\n",
            "[BWTIncCreate] textLength=6211430126, availableWord=449059136\n",
            "[BWTIncConstructFromPacked] 10 iterations done. 99999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 20 iterations done. 199999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 30 iterations done. 299999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 40 iterations done. 399999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 50 iterations done. 499999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 60 iterations done. 599999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 70 iterations done. 699999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 80 iterations done. 799999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 90 iterations done. 899999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 100 iterations done. 999999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 110 iterations done. 1099999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 120 iterations done. 1199999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 130 iterations done. 1299999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 140 iterations done. 1399999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 150 iterations done. 1499999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 160 iterations done. 1599999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 170 iterations done. 1699999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 180 iterations done. 1799999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 190 iterations done. 1899999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 200 iterations done. 1999999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 210 iterations done. 2099999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 220 iterations done. 2199999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 230 iterations done. 2299999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 240 iterations done. 2399999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 250 iterations done. 2499999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 260 iterations done. 2599999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 270 iterations done. 2699999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 280 iterations done. 2799999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 290 iterations done. 2899999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 300 iterations done. 2999999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 310 iterations done. 3099999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 320 iterations done. 3199999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 330 iterations done. 3299999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 340 iterations done. 3399999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 350 iterations done. 3499999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 360 iterations done. 3599999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 370 iterations done. 3699999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 380 iterations done. 3799999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 390 iterations done. 3899999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 400 iterations done. 3999999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 410 iterations done. 4099999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 420 iterations done. 4199999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 430 iterations done. 4299999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 440 iterations done. 4399999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 450 iterations done. 4499999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 460 iterations done. 4599999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 470 iterations done. 4699999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 480 iterations done. 4799999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 490 iterations done. 4899999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 500 iterations done. 4999999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 510 iterations done. 5099999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 520 iterations done. 5199999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 530 iterations done. 5299999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 540 iterations done. 5399999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 550 iterations done. 5499999998 characters processed.\n",
            "[BWTIncConstructFromPacked] 560 iterations done. 5597672750 characters processed.\n",
            "[BWTIncConstructFromPacked] 570 iterations done. 5685172830 characters processed.\n",
            "[BWTIncConstructFromPacked] 580 iterations done. 5762939134 characters processed.\n",
            "[BWTIncConstructFromPacked] 590 iterations done. 5832054014 characters processed.\n",
            "[BWTIncConstructFromPacked] 600 iterations done. 5893479486 characters processed.\n",
            "[BWTIncConstructFromPacked] 610 iterations done. 5948070590 characters processed.\n",
            "[BWTIncConstructFromPacked] 620 iterations done. 5996587278 characters processed.\n",
            "[BWTIncConstructFromPacked] 630 iterations done. 6039704990 characters processed.\n",
            "[BWTIncConstructFromPacked] 640 iterations done. 6078024110 characters processed.\n",
            "[BWTIncConstructFromPacked] 650 iterations done. 6112078222 characters processed.\n",
            "[BWTIncConstructFromPacked] 660 iterations done. 6142341550 characters processed.\n",
            "[BWTIncConstructFromPacked] 670 iterations done. 6169235662 characters processed.\n",
            "[BWTIncConstructFromPacked] 680 iterations done. 6193135198 characters processed.\n",
            "[bwt_gen] Finished constructing BWT in 689 iterations.\n",
            "[bwa_index] 4265.75 seconds elapse.\n",
            "[bwa_index] Update BWT... 27.96 sec\n",
            "[bwa_index] Pack forward-only FASTA... 18.06 sec\n",
            "[bwa_index] Construct SA from BWT and Occ... 2276.84 sec\n",
            "[main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa index GRCh38_no_alt_plus_decoy.fa\n",
            "[main] Real time: 6791.997 sec; CPU: 6619.048 sec\n"
          ]
        }
      ],
      "source": [
        "# ✅ Move to reference folder (just in case)\n",
        "%cd /content/reference\n",
        "\n",
        "# ✅ Create FASTA index\n",
        "!samtools faidx GRCh38_no_alt_plus_decoy.fa\n",
        "\n",
        "# ✅ Create BWA index\n",
        "!bwa index GRCh38_no_alt_plus_decoy.fa\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 16,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "JTf3DzaUkWTm",
        "outputId": "80756803-ec31-4f49-dcd7-e135481ca8b7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using GATK jar /opt/gatk/gatk-package-4.4.0.0-local.jar\n",
            "Running:\n",
            "    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /opt/gatk/gatk-package-4.4.0.0-local.jar --version\n",
            "The Genome Analysis Toolkit (GATK) v4.4.0.0\n",
            "HTSJDK Version: 3.0.5\n",
            "Picard Version: 3.0.0\n"
          ]
        }
      ],
      "source": [
        "# ✅ Install GATK (Genome Analysis Toolkit)\n",
        "!wget -q https://github.com/broadinstitute/gatk/releases/download/4.4.0.0/gatk-4.4.0.0.zip\n",
        "!unzip -q gatk-4.4.0.0.zip\n",
        "!mv gatk-4.4.0.0 /opt/gatk\n",
        "!ln -s /opt/gatk/gatk /usr/local/bin/gatk\n",
        "!gatk --version\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 17,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "N6NvpIRq-R1G",
        "outputId": "ddec810a-8f5d-44e4-c4d5-b4930579d667"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using GATK jar /opt/gatk/gatk-package-4.4.0.0-local.jar\n",
            "Running:\n",
            "    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /opt/gatk/gatk-package-4.4.0.0-local.jar CreateSequenceDictionary -R GRCh38_no_alt_plus_decoy.fa -O GRCh38_no_alt_plus_decoy.dict\n",
            "09:48:36.706 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/gatk/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "[Thu Aug 07 09:48:36 UTC 2025] CreateSequenceDictionary --OUTPUT GRCh38_no_alt_plus_decoy.dict --REFERENCE GRCh38_no_alt_plus_decoy.fa --TRUNCATE_NAMES_AT_WHITESPACE true --NUM_SEQUENCES 2147483647 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n",
            "[Thu Aug 07 09:48:36 UTC 2025] Executing as root@39a6dd3c98ca on Linux 6.1.123+ amd64; OpenJDK 64-Bit Server VM 17.0.15+6-Ubuntu-0ubuntu122.04; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n",
            "[Thu Aug 07 09:48:48 UTC 2025] picard.sam.CreateSequenceDictionary done. Elapsed time: 0.20 minutes.\n",
            "Runtime.totalMemory()=878706688\n",
            "Tool returned:\n",
            "0\n"
          ]
        }
      ],
      "source": [
        "# ✅ Create GATK sequence dictionary\n",
        "!gatk CreateSequenceDictionary \\\n",
        "   -R GRCh38_no_alt_plus_decoy.fa \\\n",
        "   -O GRCh38_no_alt_plus_decoy.dict"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 18,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FMqM_4OD-zof",
        "outputId": "73e8fd04-1106-424b-fd9d-132762b465dd"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[M::bwa_idx_load_from_disk] read 0 ALT contigs\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 80449, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.79, 44.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.020 CPU sec, 29.779 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80596, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (159.22, 44.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.361 CPU sec, 29.548 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80219, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.58, 43.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.460 CPU sec, 28.178 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80494, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.83, 44.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.788 CPU sec, 30.626 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80521, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (159.05, 44.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.126 CPU sec, 29.604 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80414, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.05, 44.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.466 CPU sec, 32.631 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80411, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.48, 43.77)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.650 CPU sec, 29.015 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80385, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.53, 44.10)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.483 CPU sec, 30.478 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80483, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.39, 44.17)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.318 CPU sec, 28.922 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80308, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.49, 44.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.509 CPU sec, 28.689 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80292, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.54, 43.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.944 CPU sec, 30.215 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80143, 0, 5)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.52, 44.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 53.040 CPU sec, 39.769 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80203, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 151, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.64, 44.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.595 CPU sec, 28.444 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80409, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.16, 43.87)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 49.240 CPU sec, 35.544 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80213, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.26, 43.87)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.422 CPU sec, 32.232 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80259, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.45, 44.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.765 CPU sec, 29.964 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80189, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.41, 43.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.947 CPU sec, 30.257 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80297, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.81, 43.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.117 CPU sec, 27.237 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80352, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (157.93, 44.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.419 CPU sec, 30.710 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79878, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.50, 43.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.043 CPU sec, 29.963 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80178, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.74, 43.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.068 CPU sec, 27.326 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80205, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.96, 43.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.688 CPU sec, 30.068 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80013, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.72, 43.25)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.248 CPU sec, 30.212 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 80161, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.22, 44.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.186 CPU sec, 30.546 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79964, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.61, 43.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.602 CPU sec, 29.892 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80016, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (157.96, 43.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.155 CPU sec, 30.089 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80328, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.46, 43.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.829 CPU sec, 27.291 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80212, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.49, 43.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.312 CPU sec, 30.484 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79881, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.41, 43.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.981 CPU sec, 28.788 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80095, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.15, 43.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.644 CPU sec, 28.235 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80027, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.36, 43.26)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.592 CPU sec, 30.581 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80165, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.13, 43.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.932 CPU sec, 28.238 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80213, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.04, 43.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.218 CPU sec, 32.407 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80070, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.99, 43.00)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.193 CPU sec, 29.256 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79996, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.84, 42.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.680 CPU sec, 33.781 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80045, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.83, 42.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.719 CPU sec, 28.946 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79877, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (157.08, 43.25)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.270 CPU sec, 31.057 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80047, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.89, 43.51)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.381 CPU sec, 29.743 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79604, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.73, 43.00)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.164 CPU sec, 28.624 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79907, 3, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.65, 43.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.972 CPU sec, 30.843 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79874, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.35, 42.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.301 CPU sec, 34.088 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80115, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.35, 42.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.744 CPU sec, 28.541 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80180, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.06, 42.60)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.171 CPU sec, 28.966 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79696, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (155.99, 42.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.370 CPU sec, 30.459 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79942, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.51, 42.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.186 CPU sec, 28.308 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80129, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.19, 42.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.891 CPU sec, 29.171 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80165, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (155.97, 42.60)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.874 CPU sec, 30.205 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80074, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.76, 42.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.687 CPU sec, 27.402 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80105, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.06, 42.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.067 CPU sec, 29.848 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79947, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.53, 42.31)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.622 CPU sec, 33.662 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80154, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.17, 42.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.588 CPU sec, 27.647 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79876, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.68, 42.12)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.119 CPU sec, 29.930 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79997, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.47, 41.86)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.027 CPU sec, 30.670 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80045, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.35, 42.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.932 CPU sec, 28.396 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80084, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.80, 44.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 49.802 CPU sec, 34.379 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79857, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.87, 44.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.251 CPU sec, 31.094 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80214, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.81, 44.65)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.441 CPU sec, 31.095 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79799, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.02, 44.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.902 CPU sec, 30.925 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80095, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.67, 44.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.057 CPU sec, 30.970 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79980, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (159.06, 44.64)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.833 CPU sec, 30.709 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80234, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.73, 44.43)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.020 CPU sec, 27.709 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80017, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.76, 44.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.389 CPU sec, 30.021 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80167, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.78, 44.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.797 CPU sec, 30.750 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80239, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.39, 44.09)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.287 CPU sec, 28.184 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80091, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.49, 44.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.400 CPU sec, 29.018 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80232, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.26, 44.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.464 CPU sec, 29.965 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80156, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.41, 44.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.441 CPU sec, 35.242 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80005, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.22, 44.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.845 CPU sec, 27.678 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79886, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.61, 43.64)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.253 CPU sec, 30.297 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 80229, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.91, 43.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.737 CPU sec, 29.282 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79765, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.68, 43.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.732 CPU sec, 27.812 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80077, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.26, 43.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.114 CPU sec, 30.310 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80018, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.56, 43.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.387 CPU sec, 29.113 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79721, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.65, 43.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.668 CPU sec, 28.704 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79920, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.86, 43.73)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.828 CPU sec, 35.337 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79913, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.73, 43.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.584 CPU sec, 31.507 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80040, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.63, 43.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.775 CPU sec, 29.641 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79892, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.58, 43.56)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.622 CPU sec, 30.158 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79907, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.59, 43.67)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.784 CPU sec, 26.979 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80081, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.37, 43.61)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.217 CPU sec, 30.464 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80079, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.31, 43.46)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.654 CPU sec, 29.303 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80163, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.51, 43.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.187 CPU sec, 28.086 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79934, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.31, 43.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.110 CPU sec, 30.552 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80089, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.99, 43.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.238 CPU sec, 32.430 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80187, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.73, 43.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.499 CPU sec, 29.952 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80078, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.11, 43.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.767 CPU sec, 30.133 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80148, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.03, 43.12)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.699 CPU sec, 29.219 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79854, 1, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.80, 43.07)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.937 CPU sec, 27.691 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80049, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.04, 43.23)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.187 CPU sec, 29.709 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80122, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.04, 43.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.099 CPU sec, 29.036 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80057, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.75, 43.23)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.220 CPU sec, 28.534 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79665, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.53, 43.05)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.931 CPU sec, 33.359 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79985, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.42, 42.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.659 CPU sec, 33.741 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79897, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.70, 43.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.394 CPU sec, 30.414 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 79987, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.51, 42.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.338 CPU sec, 33.484 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80061, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.48, 43.26)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.326 CPU sec, 28.639 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80012, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.29, 42.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.877 CPU sec, 30.848 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79952, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.70, 42.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.284 CPU sec, 29.064 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80305, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.23, 42.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.959 CPU sec, 28.680 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80123, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.04, 42.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.415 CPU sec, 30.602 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79609, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.18, 42.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.519 CPU sec, 29.861 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80067, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (155.91, 42.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.581 CPU sec, 31.757 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80137, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.19, 42.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.732 CPU sec, 29.998 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79959, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.02, 42.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.772 CPU sec, 27.077 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79890, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.27, 41.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.864 CPU sec, 30.811 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80023, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.58, 42.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.002 CPU sec, 30.403 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79599, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.06, 44.51)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.890 CPU sec, 27.979 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79726, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 188)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (5, 310)\n",
            "[M::mem_pestat] mean and std.dev: (159.44, 44.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 371)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.934 CPU sec, 30.628 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79858, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.95, 44.34)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.758 CPU sec, 30.502 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79791, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (159.13, 44.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.297 CPU sec, 30.976 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79555, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.84, 44.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.932 CPU sec, 30.708 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79727, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.85, 44.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.271 CPU sec, 30.169 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79755, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.47, 43.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.562 CPU sec, 27.308 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79943, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.91, 44.54)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.608 CPU sec, 30.601 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79774, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.39, 44.11)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.005 CPU sec, 30.677 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79942, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.50, 43.90)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.836 CPU sec, 34.530 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80009, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.52, 44.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.355 CPU sec, 27.767 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79763, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.44, 43.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.086 CPU sec, 34.259 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80053, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.44, 43.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.349 CPU sec, 29.473 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79978, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.44, 43.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.731 CPU sec, 27.959 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79971, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.63, 44.36)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.700 CPU sec, 30.089 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80091, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.55, 43.90)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.530 CPU sec, 29.243 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80092, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.20, 43.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.560 CPU sec, 28.328 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79917, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.71, 44.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.980 CPU sec, 34.716 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79963, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.23, 43.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.113 CPU sec, 28.322 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80299, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.71, 43.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.573 CPU sec, 29.270 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79896, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.79, 43.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.473 CPU sec, 33.309 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79852, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.10, 43.90)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.024 CPU sec, 27.979 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80033, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.82, 43.52)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.485 CPU sec, 29.397 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79882, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.10, 43.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.541 CPU sec, 30.603 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80009, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.70, 43.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.850 CPU sec, 28.752 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80038, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.94, 43.84)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.979 CPU sec, 29.633 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79995, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.13, 43.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.478 CPU sec, 30.412 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80087, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.67, 43.60)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.718 CPU sec, 28.979 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80000, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.76, 43.51)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.612 CPU sec, 32.247 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79825, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.59, 43.52)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.879 CPU sec, 35.091 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80208, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.94, 43.35)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.165 CPU sec, 29.479 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80045, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.85, 42.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.668 CPU sec, 28.541 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79845, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.14, 42.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.366 CPU sec, 30.543 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80304, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.52, 43.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.169 CPU sec, 28.835 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79942, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.38, 43.40)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.427 CPU sec, 29.011 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79990, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.80, 43.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.288 CPU sec, 31.044 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80028, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.02, 43.15)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.724 CPU sec, 33.843 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79950, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.10, 43.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.013 CPU sec, 27.430 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80131, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.85, 42.96)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.673 CPU sec, 30.380 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79663, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.96, 43.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.346 CPU sec, 31.152 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79441, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.89, 43.23)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.428 CPU sec, 31.324 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79871, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.87, 42.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.206 CPU sec, 32.107 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80021, 2, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.48, 42.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.857 CPU sec, 30.759 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79863, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.69, 43.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.849 CPU sec, 30.687 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79896, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.45, 42.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.282 CPU sec, 32.346 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79975, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.27, 42.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.682 CPU sec, 29.188 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79782, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.63, 43.19)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.468 CPU sec, 31.001 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79749, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.07, 42.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.008 CPU sec, 29.480 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80194, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.60, 42.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.932 CPU sec, 28.866 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79951, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.34, 42.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.038 CPU sec, 35.684 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80211, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.16, 42.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.254 CPU sec, 29.590 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80031, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.21, 42.63)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.887 CPU sec, 28.567 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 80210, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.21, 42.63)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.384 CPU sec, 34.415 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79813, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.98, 42.23)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.709 CPU sec, 30.088 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79957, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.52, 42.14)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.279 CPU sec, 28.484 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 80354, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.65, 42.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.318 CPU sec, 31.162 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79565, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.15, 44.40)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.927 CPU sec, 33.311 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79446, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.23, 44.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.899 CPU sec, 27.860 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79517, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.94, 44.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.662 CPU sec, 30.907 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79353, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.08, 44.23)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.086 CPU sec, 31.391 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79624, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.02, 44.48)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.556 CPU sec, 34.099 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79686, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.87, 44.42)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.690 CPU sec, 27.944 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79443, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.89, 44.17)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.681 CPU sec, 30.494 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79632, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.73, 44.19)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.382 CPU sec, 31.550 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 79797, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.48, 44.36)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 52.641 CPU sec, 36.150 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79797, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.37, 43.88)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.893 CPU sec, 30.076 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79595, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.82, 44.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.970 CPU sec, 28.677 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 79370, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.55, 44.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.965 CPU sec, 31.280 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79782, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.86, 44.41)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 50.231 CPU sec, 39.503 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79499, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.32, 44.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.882 CPU sec, 29.583 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79657, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.37, 44.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.777 CPU sec, 30.132 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79505, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.32, 43.96)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.739 CPU sec, 31.791 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79624, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.17, 43.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.663 CPU sec, 31.061 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79535, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.92, 43.85)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.804 CPU sec, 28.269 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79658, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.83, 43.68)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.021 CPU sec, 31.487 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79642, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.53, 43.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.459 CPU sec, 33.181 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79464, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.48, 43.29)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.430 CPU sec, 29.130 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79631, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.06, 43.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.659 CPU sec, 31.254 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79687, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.16, 43.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.858 CPU sec, 31.267 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79452, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.84, 43.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.808 CPU sec, 28.568 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79756, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.71, 43.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.345 CPU sec, 29.262 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79300, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.56, 43.61)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.991 CPU sec, 31.241 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79331, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.45, 43.52)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.932 CPU sec, 30.845 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79362, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.44, 43.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.922 CPU sec, 32.547 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79655, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.08, 42.94)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.105 CPU sec, 30.477 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79615, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.55, 43.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.303 CPU sec, 31.406 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79328, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.34, 43.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.678 CPU sec, 30.822 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79351, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.08, 42.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 55.241 CPU sec, 38.354 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79189, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.07, 43.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 50.486 CPU sec, 35.611 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79390, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.54, 43.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.149 CPU sec, 31.065 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79323, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.89, 43.31)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.403 CPU sec, 34.954 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79463, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.81, 42.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.944 CPU sec, 29.245 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79216, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.89, 43.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.156 CPU sec, 31.055 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79344, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.67, 42.96)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.841 CPU sec, 31.835 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 78969, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 297)\n",
            "[M::mem_pestat] mean and std.dev: (156.40, 42.50)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 354)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.591 CPU sec, 31.740 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79033, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.74, 43.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.056 CPU sec, 29.388 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79332, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.25, 42.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.661 CPU sec, 30.576 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79407, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.38, 42.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.202 CPU sec, 31.565 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79721, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.42, 42.63)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.525 CPU sec, 32.559 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79518, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.69, 42.93)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.695 CPU sec, 29.848 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79497, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.89, 43.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.361 CPU sec, 31.451 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79366, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.24, 42.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.959 CPU sec, 31.658 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79308, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.80, 42.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.316 CPU sec, 29.239 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79255, 3, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.38, 42.65)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.783 CPU sec, 31.875 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79563, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.69, 43.01)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.639 CPU sec, 31.087 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (2, 79660, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.71, 42.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.968 CPU sec, 34.370 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79590, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.11, 42.71)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.048 CPU sec, 28.202 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79467, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.22, 42.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 50.107 CPU sec, 35.636 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79198, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.25, 42.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.340 CPU sec, 32.154 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79382, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.82, 42.29)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.299 CPU sec, 31.487 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79070, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.00, 43.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.466 CPU sec, 29.369 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79341, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.96, 44.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.375 CPU sec, 30.170 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79406, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.69, 44.66)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.318 CPU sec, 32.172 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79135, 3, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.01, 44.38)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.219 CPU sec, 32.449 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79318, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.84, 44.41)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.007 CPU sec, 28.645 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79201, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.23, 44.05)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.101 CPU sec, 29.982 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79460, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.96, 44.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.479 CPU sec, 31.115 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79337, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.43, 44.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.900 CPU sec, 28.671 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79182, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.40, 44.33)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 51.875 CPU sec, 36.503 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79252, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.42, 43.93)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.752 CPU sec, 32.081 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79545, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.33, 44.05)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.312 CPU sec, 32.557 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79110, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.44, 44.27)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.089 CPU sec, 29.182 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79484, 3, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.23, 44.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.865 CPU sec, 30.801 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79356, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.43, 44.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.201 CPU sec, 31.133 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79591, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.18, 44.13)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.293 CPU sec, 28.590 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79343, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.43, 44.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.479 CPU sec, 30.878 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79398, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.85, 43.82)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.066 CPU sec, 31.391 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79314, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.34, 43.58)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.616 CPU sec, 39.503 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79346, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.41, 43.49)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.951 CPU sec, 28.135 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79202, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.44, 43.60)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.927 CPU sec, 29.767 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79036, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.77, 43.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.139 CPU sec, 31.376 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79394, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.55, 43.60)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.661 CPU sec, 29.634 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79495, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.25, 43.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.263 CPU sec, 28.282 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79338, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.49, 43.33)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.277 CPU sec, 30.388 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79312, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.58, 43.64)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 43.598 CPU sec, 28.633 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79324, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.39, 43.41)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.292 CPU sec, 33.280 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79544, 2, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.39, 43.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.591 CPU sec, 29.696 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79373, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (5, 305)\n",
            "[M::mem_pestat] mean and std.dev: (157.29, 43.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 365)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.036 CPU sec, 30.135 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79422, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.94, 43.14)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.811 CPU sec, 28.410 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79319, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.86, 43.21)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.644 CPU sec, 29.585 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79338, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.93, 42.91)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.232 CPU sec, 30.778 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79471, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.86, 43.02)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.725 CPU sec, 29.433 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79370, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.70, 43.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 54.020 CPU sec, 41.135 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79562, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.67, 43.18)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.037 CPU sec, 30.331 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79328, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.44, 42.93)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.311 CPU sec, 30.930 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79074, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.70, 42.76)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.107 CPU sec, 29.979 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79435, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.31, 42.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 51.600 CPU sec, 36.899 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79061, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.58, 43.08)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.365 CPU sec, 28.359 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79214, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.81, 43.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.502 CPU sec, 31.151 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79443, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.57, 43.02)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.701 CPU sec, 33.483 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79557, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.36, 43.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.602 CPU sec, 28.549 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79348, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.76, 43.04)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.062 CPU sec, 30.953 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79305, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.48, 42.78)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.403 CPU sec, 30.481 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79545, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.79, 43.22)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.068 CPU sec, 27.926 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79087, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.61, 43.16)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.708 CPU sec, 31.153 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79359, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.44, 42.89)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.724 CPU sec, 31.317 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79625, 3, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.12, 42.83)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 44.485 CPU sec, 28.360 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79410, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.32, 42.73)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.147 CPU sec, 34.655 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79502, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.03, 42.87)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.766 CPU sec, 31.005 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79261, 2, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.23, 42.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.388 CPU sec, 31.003 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79111, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.09, 42.61)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.517 CPU sec, 31.020 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79249, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (155.94, 42.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.923 CPU sec, 28.898 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79173, 1, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 182)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (11, 296)\n",
            "[M::mem_pestat] mean and std.dev: (155.86, 42.16)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 353)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.630 CPU sec, 32.354 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79369, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.80, 44.45)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.873 CPU sec, 32.372 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79435, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (159.07, 44.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.861 CPU sec, 35.437 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 78972, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.76, 44.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 59.484 CPU sec, 43.057 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79278, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.84, 44.36)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.138 CPU sec, 32.053 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79087, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.50, 43.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.859 CPU sec, 33.051 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79309, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.95, 44.74)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.609 CPU sec, 32.416 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79267, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.83, 44.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.845 CPU sec, 31.692 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79253, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (4, 309)\n",
            "[M::mem_pestat] mean and std.dev: (158.86, 44.57)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 370)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.059 CPU sec, 31.504 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79183, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.48, 44.24)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.916 CPU sec, 31.684 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79280, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.84, 44.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.489 CPU sec, 31.690 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79481, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 187)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 307)\n",
            "[M::mem_pestat] mean and std.dev: (158.82, 44.36)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 367)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.454 CPU sec, 32.053 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79368, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.57, 44.32)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.752 CPU sec, 29.101 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79516, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.58, 44.10)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.254 CPU sec, 31.358 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79319, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 152, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.52, 43.98)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.614 CPU sec, 31.668 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79377, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 304)\n",
            "[M::mem_pestat] mean and std.dev: (158.46, 43.96)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 363)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 51.220 CPU sec, 36.041 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79191, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.27, 44.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.690 CPU sec, 32.110 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79436, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.13, 43.77)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.839 CPU sec, 30.320 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79620, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.15, 43.81)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.019 CPU sec, 32.119 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79465, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 186)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (6, 306)\n",
            "[M::mem_pestat] mean and std.dev: (158.09, 44.20)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 366)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.108 CPU sec, 32.005 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79455, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.49, 43.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.337 CPU sec, 29.364 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79495, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.62, 43.70)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.599 CPU sec, 31.562 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79160, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.68, 43.47)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 50.305 CPU sec, 39.651 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79109, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.71, 43.59)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.576 CPU sec, 32.491 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79444, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.73, 43.63)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.522 CPU sec, 31.781 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79370, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.82, 43.81)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.721 CPU sec, 30.717 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79485, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.60, 43.39)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.090 CPU sec, 30.151 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79405, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.12, 43.30)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.787 CPU sec, 31.955 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79362, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.43, 43.44)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.295 CPU sec, 31.600 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79403, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 151, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.54, 43.41)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.453 CPU sec, 34.207 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79379, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.26, 43.51)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.218 CPU sec, 29.142 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79670, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 185)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (8, 303)\n",
            "[M::mem_pestat] mean and std.dev: (157.42, 43.52)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 362)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.553 CPU sec, 31.809 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 78921, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.95, 43.28)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.239 CPU sec, 31.520 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79309, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (157.06, 43.06)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 49.692 CPU sec, 32.910 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79534, 3, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.82, 43.01)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.737 CPU sec, 30.977 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79154, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.88, 43.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.350 CPU sec, 31.510 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79439, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 297)\n",
            "[M::mem_pestat] mean and std.dev: (156.46, 42.55)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 354)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.184 CPU sec, 34.637 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79355, 0, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.79, 43.03)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.295 CPU sec, 29.198 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79183, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.61, 42.69)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.011 CPU sec, 30.820 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79167, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.56, 42.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.018 CPU sec, 31.307 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79280, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.73, 43.30)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.856 CPU sec, 30.591 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 78509, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.41, 42.69)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.658 CPU sec, 30.036 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79282, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.41, 42.72)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 48.243 CPU sec, 34.809 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79456, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.30, 42.81)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.970 CPU sec, 35.887 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79228, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.16, 42.71)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 45.498 CPU sec, 30.379 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79600, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (7, 302)\n",
            "[M::mem_pestat] mean and std.dev: (156.61, 43.01)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 361)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.419 CPU sec, 29.450 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79166, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.33, 42.53)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.330 CPU sec, 31.665 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 79327, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.84, 42.99)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.274 CPU sec, 31.645 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79221, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.42, 42.95)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.087 CPU sec, 30.474 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79346, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.93, 42.92)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.434 CPU sec, 30.897 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79330, 1, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (12, 297)\n",
            "[M::mem_pestat] mean and std.dev: (156.70, 42.79)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 354)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.618 CPU sec, 35.749 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79230, 1, 2)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.34, 42.80)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.178 CPU sec, 30.833 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79455, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.27, 42.77)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.613 CPU sec, 29.285 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79119, 0, 1)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.31, 42.75)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 47.059 CPU sec, 32.099 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79461, 1, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (126, 150, 184)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (10, 300)\n",
            "[M::mem_pestat] mean and std.dev: (156.81, 42.86)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 358)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.522 CPU sec, 31.640 real sec\n",
            "[M::process] read 198020 sequences (20000020 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (1, 78988, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (155.97, 42.40)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 46.431 CPU sec, 31.087 real sec\n",
            "[M::process] read 9312 sequences (940512 bp)...\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 79312, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (125, 149, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (9, 299)\n",
            "[M::mem_pestat] mean and std.dev: (156.28, 42.54)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 357)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 198020 reads in 53.407 CPU sec, 35.996 real sec\n",
            "[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (0, 3761, 0, 0)\n",
            "[M::mem_pestat] skip orientation FF as there are not enough pairs\n",
            "[M::mem_pestat] analyzing insert size distribution for orientation FR...\n",
            "[M::mem_pestat] (25, 50, 75) percentile: (127, 150, 183)\n",
            "[M::mem_pestat] low and high boundaries for computing mean and std.dev: (15, 295)\n",
            "[M::mem_pestat] mean and std.dev: (156.85, 41.97)\n",
            "[M::mem_pestat] low and high boundaries for proper pairs: (1, 351)\n",
            "[M::mem_pestat] skip orientation RF as there are not enough pairs\n",
            "[M::mem_pestat] skip orientation RR as there are not enough pairs\n",
            "[M::mem_process_seqs] Processed 9312 reads in 2.504 CPU sec, 3.074 real sec\n",
            "[main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -t 2 /content/reference/GRCh38_no_alt_plus_decoy.fa /content/drive/MyDrive/Cancer_WES_Project/raw_fastq/SRR766039_1.fastq.gz /content/drive/MyDrive/Cancer_WES_Project/raw_fastq/SRR766039_2.fastq.gz\n",
            "[main] Real time: 10082.156 sec; CPU: 14848.052 sec\n",
            "[bam_sort_core] merging from 23 files and 1 in-memory blocks...\n",
            "Using GATK jar /opt/gatk/gatk-package-4.4.0.0-local.jar\n",
            "Running:\n",
            "    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /opt/gatk/gatk-package-4.4.0.0-local.jar AddOrReplaceReadGroups -I /content/drive/MyDrive/Cancer_WES_Project/bam_files/SRR766039_sorted.bam -O /content/drive/MyDrive/Cancer_WES_Project/bam_files/SRR766039_RG.bam -RGID 1 -RGLB lib1 -RGPL illumina -RGPU unit1 -RGSM SRR766039\n",
            "12:51:46.905 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/opt/gatk/gatk-package-4.4.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so\n",
            "[Thu Aug 07 12:51:46 UTC 2025] AddOrReplaceReadGroups --INPUT /content/drive/MyDrive/Cancer_WES_Project/bam_files/SRR766039_sorted.bam --OUTPUT /content/drive/MyDrive/Cancer_WES_Project/bam_files/SRR766039_RG.bam --RGID 1 --RGLB lib1 --RGPL illumina --RGPU unit1 --RGSM SRR766039 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_INDEX false --CREATE_MD5_FILE false --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false\n",
            "[Thu Aug 07 12:51:47 UTC 2025] Executing as root@39a6dd3c98ca on Linux 6.1.123+ amd64; OpenJDK 64-Bit Server VM 17.0.15+6-Ubuntu-0ubuntu122.04; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.4.0.0\n",
            "INFO\t2025-08-07 12:51:47\tAddOrReplaceReadGroups\tCreated read-group ID=1 PL=illumina LB=lib1 SM=SRR766039\n",
            "\n",
            "INFO\t2025-08-07 12:51:59\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:   11s.  Last read position: chr1:20,148,239\n",
            "INFO\t2025-08-07 12:52:05\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:17s.  Time for last 1,000,000:    6s.  Last read position: chr1:43,591,861\n",
            "INFO\t2025-08-07 12:52:12\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    7s.  Last read position: chr1:98,866,216\n",
            "INFO\t2025-08-07 12:52:20\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    7s.  Last read position: chr1:148,121,986\n",
            "INFO\t2025-08-07 12:52:25\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:38s.  Time for last 1,000,000:    5s.  Last read position: chr1:161,000,743\n",
            "INFO\t2025-08-07 12:52:34\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:47s.  Time for last 1,000,000:    9s.  Last read position: chr1:205,065,179\n",
            "INFO\t2025-08-07 12:52:40\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:    5s.  Last read position: chr1:247,606,339\n",
            "INFO\t2025-08-07 12:52:49\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    9s.  Last read position: chr2:45,842,963\n",
            "INFO\t2025-08-07 12:52:55\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:01:07s.  Time for last 1,000,000:    6s.  Last read position: chr2:96,025,429\n",
            "INFO\t2025-08-07 12:53:01\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:01:14s.  Time for last 1,000,000:    6s.  Last read position: chr2:131,177,537\n",
            "INFO\t2025-08-07 12:53:10\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:01:22s.  Time for last 1,000,000:    8s.  Last read position: chr2:181,918,505\n",
            "INFO\t2025-08-07 12:53:15\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:28s.  Time for last 1,000,000:    5s.  Last read position: chr2:232,481,471\n",
            "INFO\t2025-08-07 12:53:25\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:37s.  Time for last 1,000,000:    9s.  Last read position: chr3:39,079,739\n",
            "INFO\t2025-08-07 12:53:30\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:43s.  Time for last 1,000,000:    5s.  Last read position: chr3:66,380,617\n",
            "INFO\t2025-08-07 12:53:38\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    7s.  Last read position: chr3:134,951,415\n",
            "INFO\t2025-08-07 12:53:45\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:58s.  Time for last 1,000,000:    7s.  Last read position: chr3:197,830,924\n",
            "INFO\t2025-08-07 12:53:51\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    5s.  Last read position: chr4:70,158,594\n",
            "INFO\t2025-08-07 12:54:00\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:02:13s.  Time for last 1,000,000:    9s.  Last read position: chr4:150,215,635\n",
            "INFO\t2025-08-07 12:54:06\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:02:18s.  Time for last 1,000,000:    5s.  Last read position: chr5:41,313,576\n",
            "INFO\t2025-08-07 12:54:15\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:02:28s.  Time for last 1,000,000:    9s.  Last read position: chr5:124,646,996\n",
            "INFO\t2025-08-07 12:54:21\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:02:33s.  Time for last 1,000,000:    5s.  Last read position: chr5:168,200,019\n",
            "INFO\t2025-08-07 12:54:28\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:02:40s.  Time for last 1,000,000:    7s.  Last read position: chr6:30,670,856\n",
            "INFO\t2025-08-07 12:54:36\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:48s.  Time for last 1,000,000:    7s.  Last read position: chr6:65,788,716\n",
            "INFO\t2025-08-07 12:54:41\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:54s.  Time for last 1,000,000:    5s.  Last read position: chr6:150,935,242\n",
            "INFO\t2025-08-07 12:54:51\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:03:03s.  Time for last 1,000,000:    9s.  Last read position: chr7:34,920,022\n",
            "INFO\t2025-08-07 12:54:56\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:03:09s.  Time for last 1,000,000:    5s.  Last read position: chr7:89,333,527\n",
            "INFO\t2025-08-07 12:55:05\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:03:17s.  Time for last 1,000,000:    8s.  Last read position: chr7:139,525,403\n",
            "INFO\t2025-08-07 12:55:11\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:03:24s.  Time for last 1,000,000:    6s.  Last read position: chr8:25,364,655\n",
            "INFO\t2025-08-07 12:55:18\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:03:30s.  Time for last 1,000,000:    6s.  Last read position: chr8:118,110,136\n",
            "INFO\t2025-08-07 12:55:26\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:03:39s.  Time for last 1,000,000:    8s.  Last read position: chr9:35,740,494\n",
            "INFO\t2025-08-07 12:55:32\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:03:45s.  Time for last 1,000,000:    5s.  Last read position: chr9:105,521,043\n",
            "INFO\t2025-08-07 12:55:41\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:03:54s.  Time for last 1,000,000:    9s.  Last read position: chr9:137,100,389\n",
            "INFO\t2025-08-07 12:55:47\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:04:00s.  Time for last 1,000,000:    5s.  Last read position: chr10:48,890,577\n",
            "INFO\t2025-08-07 12:55:56\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:04:09s.  Time for last 1,000,000:    8s.  Last read position: chr10:99,804,183\n",
            "INFO\t2025-08-07 12:56:02\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:04:15s.  Time for last 1,000,000:    6s.  Last read position: chr11:4,594,264\n",
            "INFO\t2025-08-07 12:56:09\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:04:21s.  Time for last 1,000,000:    6s.  Last read position: chr11:56,613,214\n",
            "INFO\t2025-08-07 12:56:17\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:04:30s.  Time for last 1,000,000:    8s.  Last read position: chr11:77,229,175\n",
            "INFO\t2025-08-07 12:56:23\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:04:35s.  Time for last 1,000,000:    5s.  Last read position: chr11:129,451,091\n",
            "INFO\t2025-08-07 12:56:32\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:04:45s.  Time for last 1,000,000:    9s.  Last read position: chr12:46,364,684\n",
            "INFO\t2025-08-07 12:56:38\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:04:50s.  Time for last 1,000,000:    5s.  Last read position: chr12:81,151,961\n",
            "INFO\t2025-08-07 12:56:46\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:04:59s.  Time for last 1,000,000:    8s.  Last read position: chr12:132,142,646\n",
            "INFO\t2025-08-07 12:56:53\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:05:05s.  Time for last 1,000,000:    6s.  Last read position: chr13:104,177,735\n",
            "INFO\t2025-08-07 12:56:59\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:05:11s.  Time for last 1,000,000:    5s.  Last read position: chr14:61,864,619\n",
            "INFO\t2025-08-07 12:57:08\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:05:21s.  Time for last 1,000,000:    9s.  Last read position: chr14:106,627,236\n",
            "INFO\t2025-08-07 12:57:14\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:05:26s.  Time for last 1,000,000:    5s.  Last read position: chr15:48,738,963\n",
            "INFO\t2025-08-07 12:57:23\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:05:36s.  Time for last 1,000,000:    9s.  Last read position: chr15:86,331,010\n",
            "INFO\t2025-08-07 12:57:29\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:05:41s.  Time for last 1,000,000:    5s.  Last read position: chr16:17,018,693\n",
            "INFO\t2025-08-07 12:57:37\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:05:49s.  Time for last 1,000,000:    7s.  Last read position: chr16:56,362,932\n",
            "INFO\t2025-08-07 12:57:44\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:05:56s.  Time for last 1,000,000:    7s.  Last read position: chr17:1,661,689\n",
            "INFO\t2025-08-07 12:57:50\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:06:02s.  Time for last 1,000,000:    5s.  Last read position: chr17:19,158,840\n",
            "INFO\t2025-08-07 12:57:59\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:06:11s.  Time for last 1,000,000:    9s.  Last read position: chr17:42,823,720\n",
            "INFO\t2025-08-07 12:58:04\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:06:17s.  Time for last 1,000,000:    5s.  Last read position: chr17:74,356,492\n",
            "INFO\t2025-08-07 12:58:14\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:06:27s.  Time for last 1,000,000:    9s.  Last read position: chr18:46,240,205\n",
            "INFO\t2025-08-07 12:58:20\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:06:33s.  Time for last 1,000,000:    5s.  Last read position: chr19:9,381,520\n",
            "INFO\t2025-08-07 12:58:27\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:06:40s.  Time for last 1,000,000:    7s.  Last read position: chr19:37,489,722\n",
            "INFO\t2025-08-07 12:58:35\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:06:47s.  Time for last 1,000,000:    7s.  Last read position: chr19:52,511,249\n",
            "INFO\t2025-08-07 12:58:41\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:06:53s.  Time for last 1,000,000:    5s.  Last read position: chr20:32,797,267\n",
            "INFO\t2025-08-07 12:58:50\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:07:03s.  Time for last 1,000,000:    9s.  Last read position: chr21:25,683,859\n",
            "INFO\t2025-08-07 12:58:56\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:07:08s.  Time for last 1,000,000:    5s.  Last read position: chr22:24,244,567\n",
            "INFO\t2025-08-07 12:59:05\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:07:18s.  Time for last 1,000,000:    9s.  Last read position: chrX:1,206,434\n",
            "INFO\t2025-08-07 12:59:11\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:07:23s.  Time for last 1,000,000:    5s.  Last read position: chrX:48,384,021\n",
            "INFO\t2025-08-07 12:59:17\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:07:30s.  Time for last 1,000,000:    6s.  Last read position: chrX:73,769,783\n",
            "INFO\t2025-08-07 12:59:25\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:07:38s.  Time for last 1,000,000:    8s.  Last read position: chrX:120,544,606\n",
            "INFO\t2025-08-07 12:59:31\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:07:44s.  Time for last 1,000,000:    5s.  Last read position: chrX:154,929,743\n",
            "[Thu Aug 07 12:59:35 UTC 2025] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 7.81 minutes.\n",
            "Runtime.totalMemory()=243269632\n",
            "Tool returned:\n",
            "0\n",
            "total 16G\n",
            "-rw------- 1 root root 6.1G Aug  7 12:37 SRR766039_raw.bam\n",
            "-rw------- 1 root root 5.5G Aug  7 12:59 SRR766039_RG.bam\n",
            "-rw------- 1 root root 4.1M Aug  7 13:01 SRR766039_RG.bam.bai\n",
            "-rw------- 1 root root 4.2G Aug  7 12:51 SRR766039_sorted.bam\n"
          ]
        }
      ],
      "source": [
        "# ✅ Final, clean alignment to local reference\n",
        "sample_id = \"SRR766039\"\n",
        "\n",
        "fq1 = f\"/content/drive/MyDrive/Cancer_WES_Project/raw_fastq/{sample_id}_1.fastq.gz\"\n",
        "fq2 = f\"/content/drive/MyDrive/Cancer_WES_Project/raw_fastq/{sample_id}_2.fastq.gz\"\n",
        "ref = \"/content/reference/GRCh38_no_alt_plus_decoy.fa\"\n",
        "bam_dir = \"/content/drive/MyDrive/Cancer_WES_Project/bam_files\"\n",
        "\n",
        "raw_bam = f\"{bam_dir}/{sample_id}_raw.bam\"\n",
        "sorted_bam = f\"{bam_dir}/{sample_id}_sorted.bam\"\n",
        "rg_bam = f\"{bam_dir}/{sample_id}_RG.bam\"\n",
        "\n",
        "!mkdir -p {bam_dir}\n",
        "\n",
        "# ✅ Step 1: BWA MEM\n",
        "!bwa mem -t 2 {ref} {fq1} {fq2} | samtools view -Sb - > {raw_bam}\n",
        "\n",
        "# ✅ Step 2: Sort\n",
        "!samtools sort -o {sorted_bam} {raw_bam}\n",
        "\n",
        "# ✅ Step 3: Add read groups\n",
        "!gatk AddOrReplaceReadGroups \\\n",
        "    -I {sorted_bam} \\\n",
        "    -O {rg_bam} \\\n",
        "    -RGID 1 -RGLB lib1 -RGPL illumina -RGPU unit1 -RGSM {sample_id}\n",
        "\n",
        "# ✅ Step 4: Index BAM\n",
        "!samtools index {rg_bam}\n",
        "\n",
        "# ✅ Step 5: Check final BAMs\n",
        "!ls -lh {bam_dir}\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 19,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "a9WpCyVnD1-U",
        "outputId": "f141bdde-0505-4ae5-a3d1-11a5cb89e9ea"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid\n",
            "[mpileup] 1 samples in 1 input files\n",
            "[mpileup] maximum number of reads per input file set to -d 250\n",
            "total 78M\n",
            "-rw------- 1 root root 11M Aug  7 13:55 SRR766039_raw.bcf\n",
            "-rw------- 1 root root 68M Aug  7 13:55 SRR766039_raw.vcf\n"
          ]
        }
      ],
      "source": [
        "vcf_dir = \"/content/drive/MyDrive/Cancer_WES_Project/vcf_files\"\n",
        "bcf_file = f\"{vcf_dir}/{sample_id}_raw.bcf\"\n",
        "vcf_file = f\"{vcf_dir}/{sample_id}_raw.vcf\"\n",
        "\n",
        "# Generate BCF (binary variant file)\n",
        "!bcftools mpileup -Ou -f {ref} {rg_bam} | bcftools call -mv -Ob -o {bcf_file}\n",
        "\n",
        "# Convert BCF to human-readable VCF\n",
        "!bcftools view {bcf_file} -Ov -o {vcf_file}\n",
        "\n",
        "# List result\n",
        "!ls -lh {vcf_dir}\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "HNSkP3Ml4NAr"
      },
      "source": []
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "import io\n",
        "\n",
        "# Path to your VCF file\n",
        "vcf_path = \"/content/drive/MyDrive/Cancer_WES_Project/vcf_files/SRR766039_raw.vcf\"\n",
        "\n",
        "# Read the file, skipping meta lines (starting with ##)\n",
        "with open(vcf_path, 'r') as f:\n",
        "    lines = [line for line in f if not line.startswith('##')]\n",
        "\n",
        "# Now use io.StringIO instead of pandas.compat\n",
        "vcf_df = pd.read_csv(io.StringIO(''.join(lines)), sep='\\t')\n",
        "\n",
        "# Show first few rows\n",
        "vcf_df.head()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "oz8WC8w_0_y5",
        "outputId": "53b1ab59-260d-4a87-cb44-2ada2032dbe1"
      },
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "  #CHROM    POS ID REF ALT       QUAL FILTER  \\\n",
              "0   chr1  10150  .   C   T    5.00874      .   \n",
              "1   chr1  10321  .   C   T    5.04598      .   \n",
              "2   chr1  14653  .   C   T  129.00700      .   \n",
              "3   chr1  14677  .   G   A  207.08800      .   \n",
              "4   chr1  16257  .   G   C    8.88822      .   \n",
              "\n",
              "                                                INFO FORMAT     SRR766039  \n",
              "0  DP=4;SGB=-0.379885;RPBZ=1.22474;MQBZ=1.41421;M...  GT:PL    1/1:30,0,0  \n",
              "1  DP=1;SGB=-0.379885;FS=0;MQ0F=0;AC=2;AN=2;DP4=0...  GT:PL    1/1:33,3,0  \n",
              "2  DP=55;VDB=0.0024534;SGB=-0.693127;RPBZ=-0.7989...  GT:PL  1/1:156,11,0  \n",
              "3  DP=78;VDB=0.00065505;SGB=-0.693147;RPBZ=-2.881...  GT:PL  1/1:234,32,0  \n",
              "4  DP=5;VDB=0.0221621;SGB=-0.511536;RPBZ=-1.73205...  GT:PL   0/1:41,0,33  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-019ef499-ef2b-4d3d-ac14-87521bec98a7\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>#CHROM</th>\n",
              "      <th>POS</th>\n",
              "      <th>ID</th>\n",
              "      <th>REF</th>\n",
              "      <th>ALT</th>\n",
              "      <th>QUAL</th>\n",
              "      <th>FILTER</th>\n",
              "      <th>INFO</th>\n",
              "      <th>FORMAT</th>\n",
              "      <th>SRR766039</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>chr1</td>\n",
              "      <td>10150</td>\n",
              "      <td>.</td>\n",
              "      <td>C</td>\n",
              "      <td>T</td>\n",
              "      <td>5.00874</td>\n",
              "      <td>.</td>\n",
              "      <td>DP=4;SGB=-0.379885;RPBZ=1.22474;MQBZ=1.41421;M...</td>\n",
              "      <td>GT:PL</td>\n",
              "      <td>1/1:30,0,0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>chr1</td>\n",
              "      <td>10321</td>\n",
              "      <td>.</td>\n",
              "      <td>C</td>\n",
              "      <td>T</td>\n",
              "      <td>5.04598</td>\n",
              "      <td>.</td>\n",
              "      <td>DP=1;SGB=-0.379885;FS=0;MQ0F=0;AC=2;AN=2;DP4=0...</td>\n",
              "      <td>GT:PL</td>\n",
              "      <td>1/1:33,3,0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>chr1</td>\n",
              "      <td>14653</td>\n",
              "      <td>.</td>\n",
              "      <td>C</td>\n",
              "      <td>T</td>\n",
              "      <td>129.00700</td>\n",
              "      <td>.</td>\n",
              "      <td>DP=55;VDB=0.0024534;SGB=-0.693127;RPBZ=-0.7989...</td>\n",
              "      <td>GT:PL</td>\n",
              "      <td>1/1:156,11,0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>chr1</td>\n",
              "      <td>14677</td>\n",
              "      <td>.</td>\n",
              "      <td>G</td>\n",
              "      <td>A</td>\n",
              "      <td>207.08800</td>\n",
              "      <td>.</td>\n",
              "      <td>DP=78;VDB=0.00065505;SGB=-0.693147;RPBZ=-2.881...</td>\n",
              "      <td>GT:PL</td>\n",
              "      <td>1/1:234,32,0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>chr1</td>\n",
              "      <td>16257</td>\n",
              "      <td>.</td>\n",
              "      <td>G</td>\n",
              "      <td>C</td>\n",
              "      <td>8.88822</td>\n",
              "      <td>.</td>\n",
              "      <td>DP=5;VDB=0.0221621;SGB=-0.511536;RPBZ=-1.73205...</td>\n",
              "      <td>GT:PL</td>\n",
              "      <td>0/1:41,0,33</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-019ef499-ef2b-4d3d-ac14-87521bec98a7')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-019ef499-ef2b-4d3d-ac14-87521bec98a7 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-019ef499-ef2b-4d3d-ac14-87521bec98a7');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-55c20471-5834-4a16-ad1c-6281d8e7f64d\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-55c20471-5834-4a16-ad1c-6281d8e7f64d')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-55c20471-5834-4a16-ad1c-6281d8e7f64d button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "vcf_df"
            }
          },
          "metadata": {},
          "execution_count": 21
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "vcf_df.to_csv(\"/content/drive/MyDrive/Cancer_WES_Project/vcf_files/SRR766039_raw.csv\", index=False)\n"
      ],
      "metadata": {
        "id": "TqGjvs711QXU"
      },
      "execution_count": 22,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import files\n",
        "\n",
        "# Provide the full path to your CSV file\n",
        "files.download(\"/content/drive/MyDrive/Cancer_WES_Project/vcf_files/SRR766039_raw.csv\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "id": "XXaTOnu11aNI",
        "outputId": "2abb8e6c-9377-49df-f04f-a4a84f36d46e"
      },
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_e21211b5-7d31-4d0c-aef8-cb6d18cc5da4\", \"SRR766039_raw.csv\", 72868112)"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import files\n",
        "\n",
        "# Download VCF (main file for annotation)\n",
        "files.download(\"/content/drive/MyDrive/Cancer_WES_Project/vcf_files/SRR766039_raw.vcf\")\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        },
        "id": "QjSSO6E44P8O",
        "outputId": "2655713d-5524-4c32-b1f3-15b8e57997b5"
      },
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_e020bfaf-ea92-4e3f-a41a-ca9ff440bb7f\", \"SRR766039_raw.vcf\", 70652404)"
            ]
          },
          "metadata": {}
        }
      ]
    }
  ],
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}